content/en/docs/migration/flux-guide.md

```diff
@@ -18,76 +19,47 @@ accounting for the fact that it's a system with a substantially different
 design.
 This may at times mean that you have to make adjustments to the way your
 current cluster configuration is structured. If you are in this situation
-and need help, please refer to the [support page](https://fluxcd.io/support/).
+and need help, refer to the [support page](https://fluxcd.io/support/).
 {{% /alert %}}
 
 ## Prerequisites
 
-You will need a Kubernetes cluster version **1.16** or newer
-and kubectl version **1.18** or newer.
-
-### Install Flux v2 CLI
-
-With Homebrew:
-
-```sh
-brew install fluxcd/tap/flux
-```
-
-With Bash:
-
-```sh
-curl -s https://fluxcd.io/install.sh | sudo bash
-
-# enable completions in ~/.bash_profile
-. <(flux completion bash)
-```
-
-Command-line completion for `zsh`, `fish`, and `powershell`
-are also supported with their own sub-commands.
-
-Binaries for macOS, Windows and Linux AMD64/ARM are available for download on the
-[release page](https://github.com/fluxcd/flux2/releases).
-
-Verify that your cluster satisfies the prerequisites with:
-
-```sh
-flux check --pre
-```
+- Flux CLI
+- Kubernetes cluster version **1.16** or newer
+- ``kubectl`` version **1.18** or newer.
 
 ## GitOps migration
 
+With this method, you follow the usual bootstrap procedure, which creates a new git repository, then move your manifests over.
+
 Flux v2 offers an installation procedure that is declarative first
 and disaster resilient.
 
-Using the `flux bootstrap` command you can install Flux on a
-Kubernetes cluster and configure it to manage itself from a Git
-repository. The Git repository created during bootstrap can be used 
-to define the state of your fleet of Kubernetes clusters.
+Using the `flux bootstrap` command you can install Flux on a Kubernetes cluster and configure it to manage itself from a Git repository. The Git repository created during bootstrap can be used to define the state of your fleet of Kubernetes clusters.
 
-For a detailed walk-through of the bootstrap procedure please see the [installation
+For a detailed walk-through of the bootstrap procedure see the [installation
 guide](../installation/_index.md).
 
 {{% alert color="info" color="warning" title="'flux bootstrap' target" %}}
-`flux bootstrap` should not be run against a Git branch or path
+`flux bootstrap` must not be run against a Git branch or path
 that is already being synchronized by Flux v1, as this will make
-them fight over the resources. Instead, bootstrap to a **new Git
+them fight over the resources. Bootstrap to a **new Git
 repository, branch or path**, and continue with moving the
 manifests.
 {{% /alert %}}
 
-After you've installed Flux v2 on your cluster using bootstrap,
-you can delete the Flux v1 from your clusters and move the manifests from the
-Flux v1 repository to the bootstrap one.
+After installing Flux v2 on your cluster using bootstrap, you can delete the Flux v1 from your clusters and move the manifests from the Flux v1 repository to the bootstrap one.
```

Comment:
Logically this doesn't make sense

- What the method involves doing
```
With this method, you follow the standard bootstrap steps to install flux on your cluster. After following the bootstrap you then move your manifests over to the repository that was created during bootstrap.
```
- How to do the method
```
- run the bootstrap, create new repo
- warn for migration on a branch/path already sync
- remove flux v1
- migrate manifests from v1 repo to v2 repo
```
- Further reading
```
- Flux v1 Bootstrap doc
```